<ns0:entry xmlns:ns0="http://www.w3.org/2005/Atom" xmlns:ns1="http://schemas.google.com/g/2005" xmlns:ns2="http://schemas.google.com/projecthosting/issues/2009" ns1:etag="W/&quot;AkQFQ347eCl7ImA9WhdSEko.&quot;">
		<ns0:id>http://code.google.com/feeds/issues/p/nativeclient/issues/2043/comments/full/11</ns0:id><ns0:author>
			<ns0:name>mball@google.com</ns0:name><ns0:uri>/u/mball@google.com/</ns0:uri></ns0:author><ns0:content type="html">We've been getting lots of py_auto flakiness recently on the mac 10.5 trybot, where it fails during the tab stress test with an error like this:


*** [PPB_Graphics2D::Stress BEGIN]
*** [PPB_Graphics2D::Stress FAIL] ('TestStress:ERROR at tests/ppapi_browser/ppb_graphics2d/ppapi_ppb_graphics2d.cc:522: graphics2d[i] != kInvalidResource
*** ' != 'TestStress:PASSED')

Might this be a related issue with memory allocation?</ns0:content><ns0:updated>2011-07-21T19:58:32.000Z</ns0:updated><ns0:published>2011-07-21T19:58:32.000Z</ns0:published><ns2:updates /><ns0:title>Comment 11 by mball@google.com</ns0:title><ns0:link href="http://code.google.com/p/nativeclient/issues/detail?id=2043#c11" rel="alternate" type="text/html" /><ns0:link href="https://code.google.com/feeds/issues/p/nativeclient/issues/2043/comments/full/11" rel="self" type="application/atom+xml" /></ns0:entry>